File: leetcode_notes/leetcode/0080.删除排序数组中的重复项II/0080-删除排序数组中的重复项II.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums == null) return 0;
        if(nums.length <= 2) return nums.length;
        
        int rtn = 2;
        for(int i = 2; i < nums.length; i++) {
            if(nums[i] != nums[rtn - 2]) {
                nums[rtn++] = nums[i];
            }
        }
        return rtn;
    }
}

File: leetcode_notes/leetcode_cn/0080.删除排序数组中的重复项II/0080-删除排序数组中的重复项II.c
int removeDuplicates(int* nums, int numsSize) {
int i = 2;
int j = 2;
int a,b;
a = nums[0];
b = nums[1];
if(numsSize < 3) return numsSize;
while(i < numsSize){
    if(nums[i] == a && nums[i] == b){
        i++;
        continue;
    }
    else{
        a = nums[i - 1];
        b = nums[i];
        nums[j] = nums[i];
        j++;
    }
    i++;
}
return j;

}